File: abc324/b/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	var flag bool = false
	fmt.Scanf("%d", &n)

	for i := range make([]struct{}, n) {
		for j := 0; j < n; j++ {
			resolv := int(math.Pow(2, float64(i))) * int(math.Pow(3, float64(j)))
			if resolv == n {
				flag = true
				break
			}
			fmt.Printf("x: %d, y: %d resolv: %d\n", i, j, resolv)
		}
		if flag {
			break
		}
	}

	if flag {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

File: abc324/b/main.c
#include <stdio.h>
#include <math.h>
#include <stdbool.h>

int main() {
    int n;
    bool flag = false;
    scanf("%d", &n);

    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++){
            int resolv = (int)pow((double)2, (double)i) * (int)pow((double)3, (double)j);
            if(resolv == n) {
                flag = true;
                break;
            }
        }
        if(flag){
            break;
        }
    }
    if(flag){
        printf("Yes\n");
    } else {
        printf("No\n");
    }
    return 0;
}